Add script to Wasabi on Tails setup (#249)
zkSNACKs/WasabiDoc: docs/using-wasabi/WasabiSetupTails.md

:::danger
Remember to do backups!
:::

## Script to automatically install Wasabi on Tails

> Alternatively, you can use this script made by [permabull](https://github.com/permabull/wasabi_tails_installer/blob/master/wasabi_tails_installer), which, after downloading Wasabi by following [step 2](/using-wasabi/WasabiSetupTails.html#download), automatically installs Wasabi from the persistent folder and moves the wallet you wanna open (or all of them) by userinput:

Reviewer: ```suggestion
Alternatively, you can use this [script](https://github.com/permabull/wasabi_tails_installer/blob/master/wasabi_tails_installer) made by [permabull](https://github.com/permabull), which, after downloading Wasabi by following [step 2](/using-wasabi/WasabiSetupTails.html#download), automatically installs Wasabi from the persistent folder and moves the wallet you wanna open (or all of them) by user input:
```

```
#!/bin/bash

> sudo dpkg -i Wasabi-1.1.9.2.deb

Reviewer 2: Do not use specific versions, it will be impossible to keep the documentation updated.

```suggestion
sudo dpkg -i Wasabi-<version>.deb
```

Reviewer 3: We have the `${currentVersion}` variable now, see #268.

wassabee </dev/null &>/dev/null &

sleep 5s

pkill wassabee

echo "*********************"

ls -1 -d */

echo "*********************"

while true
do	
    read -p "Enter wallet to open: " wallet_name
    FOLDER="$wallet_name"

    if [ -d "$FOLDER" ]
    then
        echo "$FOLDER wallet found."
	cd "$FOLDER"/.walletwasabi/
	cp -r client/* ~/.walletwasabi/client

> 	echo "Your files has been moved to wasabi folder"

Reviewer: ```suggestion
	echo "Your files have been moved to wasabi folder"
```

	break
    else

> 	echo ""$FOLDER" wallet dont exist"

Reviewer: ```suggestion
	echo ""$FOLDER" wallet doesn't exist"
```
